app/src/main/java/org/mozilla/fenix/historymetadata/view/RecentlyVisited.kt

```diff
@@ -213,17 +264,79 @@ private fun RecentlyVisitedCaption(count: Int) {
     )
 }
 
+@Composable
+private fun RecentlyVisitedMenu(
```

Comment:
nit: Add kDoc

Reply:
Updated in https://github.com/mozilla-mobile/fenix/pull/22535.